Request for code review

patriciachrysy opened the pull request:
Dear TSE,
i used the weather api for the weather infos and the gif API for the animated images. I separated the logic from the dom actions, give the temperatures in celsius and kelvin and made sure that the appearance changes according to the provision in the researched city.
Thanks for reviewing

Oghenebrume50 commented on src/api_handler.js:
```diff
@@ -0,0 +1,60 @@
+/*
+  eslint-disable no-unused-vars, no-alert, no-mixed-operators, import/prefer-default-export,
+  no-empty
+*/
```
are all these warnings actually thrown by eslint or you just copied these into most of your files. For examples, I do not see why  "import/prefer-defautt-export" would come up when you are exporting multiple modules and not importing anything at all  :thinking:

Oghenebrume50 commented on src/api_handler.js:
```diff
@@ -0,0 +1,60 @@
+        farenheit: ((data.main.temp_max - 273.15) * 9 / 5 + 32).toFixed(1),
+      },
+    };
+  } catch (error) {
```
Why is the catch block empty, it should be doing something to handle any error that occurs here

Oghenebrume50 commented on src/app_logic.js:
```diff
@@ -0,0 +1,58 @@
+
+    listenSearch();
+  } catch (error) {
+    alert(error);
```
Just alerting the actual error to the UI isn't really a good user experience, you do not expect your users to know how to read these errors

Oghenebrume50 commented on src/app_logic.js:
```diff
@@ -0,0 +1,58 @@
+  }
+};
+
+export { loadDefaultDatas };
```
it seems you are exporting just one function in this file, how about using export default and reduce the commented out warnings at the top

Oghenebrume50 commented on src/form_handler.js:
```diff
@@ -0,0 +1,13 @@
+  return null;
+};
+
+export { handleForm };
```
export default works here too

Also use the rules that are necessary in each file, all the commented out rules here do not look like an issue for this file at all, once you correct the export default

Oghenebrume50 commented on src/rigth_side_dom.js:
```diff
@@ -0,0 +1,34 @@
+  return wrapper;
+};
+
+export { rigthSide };
```
export default

Oghenebrume50 reviewed (changes requested):
### CHANGES REQUIRED

Hi, @patriciachrysy good job working on this, you have done amazingly well, the UI looks great and the app works amazingly well. However, I would like to point out a few things, that should be changed and/or upgraded to work better. 

1. This shouldn't display just the error type, users cannot read this error, "city not found" is good enough or maybe changing the page to show a not found background, whatever works for you just do not dump the interpreter error back to the user.
![error](https://user-images.githubusercontent.com/37092867/92686583-dcb4b980-f331-11ea-8286-0271b923a13e.png)

2. You want to create a toggle for the temperature unit conversion, do not show both celsius and Fahrenheit at once.
![see](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png)

Please make changes and request for another review, when done. 

**RAPHAEL NORIODE** :mask: STAY_SAFE

patriciachrysy commented:
hey TSE,
applied the requested changes:
- added temperature unit button controls
- pop better errors
- corrected the imports and exports
- removed useless linter deactivation
Thanks for reviewing again

mosaaleb reviewed (approved):
#### Well done :clap:

You can merge :+1: 

##### Muhammad Ebeid
